app/assets/stylesheets/alchemy/base.scss

```diff
@@ -339,11 +339,11 @@ div.all_rights {
 }
 
 .date_select_without_time #date_date_4i {
-  display: none;
+  @extend .hidden;
 }
 
 .date_select_without_time #date_date_5i {
-  display: none;
+  @extend .hidden;
 }
```

Comment:
I think that the extends do not add value, but only introduce a level of indirection for the reader (me). They would've been nice if we were to ever change the definition of `hidden` to something else more than `display: none`, but that won't ever happen as being `extended` in so many places would actually make the effect unpredictable.